pages/en/contrib/code-contrib.md

```diff
@@ -80,6 +80,20 @@ All LoopBack projects follow the same convention for running tests. To run a pro
 npm test
 ```
 
+## Create example repositories
+
+When you create an example repository for any usage demo, the name of repository should follow the following convention:
+
+`loopback{major_version}-example-{example_name}`
+
+Usually the `{major_version}` is the lastet major version under development.
+If the example aims at a specific version of LoopBack, then use that number as the `{major_version}`.
+The `{example_name}` is a few words briefly describes the functionality of the example,
```

Comment:
I think we should say `is one or two words which briefly describe` here instead of `few`. IMO, Anything more than two would be long.

Reply:
@b-admike we have examples with 3 words :( http://loopback.io/doc/en/lb3/Tutorials-and-examples.html#topic-specific-examples

Reply:
oh okay :-)